src/applications/third-party-app-directory/sass/index.scss

```diff
@@ -3,6 +3,13 @@
 }
 
 .third-party-app-directory {
+  h2 {
+    font-size: 1.17em;
```

Comment:
might be able to use the `font-size` [utilities](https://design.va.gov/utilities/font-size)